src/schemaCompare/schemaCompareWebViewController.ts

```diff
@@ -209,7 +230,39 @@ export class SchemaCompareWebViewController extends ReactWebviewPanelController<
             this.logger.verbose(`No source context provided - OperationId: ${this.operationId}`);
         }
 
-        await this.launch(source, undefined, false, comparisonResult);
+        // Resolve target (same rules as source)
```

Comment:
If it's the same rules of source, this should be a helper method.  When you move this to a helper method, please clearly comment the different cases (types of input that sourceContext might be).